server/server.js

```diff
@@ -0,0 +1,17 @@
+const path = require('path');
```

Comment:
What is the point of this empty server? Are you planning to use API on Node.js?

Reply:
I'm using it on Heroku. About API, I'm not quite sure now.